src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -152,7 +152,7 @@ private static String buildWholeModuleExportSymbolName(String importedModuleId)
    * https://github.com/angular/clutz/issues/596
    */
   private static String buildNamedExportSymbolName(String importedModuleId, String originalName) {
-    return "module$exports$" + importedModuleId.replace(".", "$") + "_" + originalName;
+    return "module$exports$" + importedModuleId.replace(".", "$") + "." + originalName;
```

Comment:
does this need a change in tsickle?

Reply:
Yes - https://github.com/angular/tsickle/pull/671